### PR TITLE
fixes the issue #439 where the status is not shown and gives a warning message that the field is invalid

### PR DESCRIPTION
**Pull request by anshulvermapatel**
Team,

I created [this issue](https://github.com/openshift/cluster-logging-operator/issues/439) and created this PR to have the correct path to get the status of ES and Kibana.

Do let me know if any modification is required.

### CODE REVIEW

**Comment by ewolinetz** [comment]
@anshulvermapatel can you please change the target of this to `master` instead? We will then cherry-pick backwards

File: manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -418,27 +418,27 @@ spec:
       statusDescriptors:
       - description: The status for each of the Kibana pods for the Visualization component
         displayName: Kibana Status
-        path: visualization.kibanaStatus.pods
+        path: visualization.kibanaStatus[0].pods
```
Comment (jcantrill) [inline manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml]: What happens when there is more then one status?  What happens when there is no status?

**Comment by jcantrill** [comment]
> @anshulvermapatel can you please change the target of this to `master` instead? We will then cherry-pick backwards

I rebased it on master

File: manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml (hunk 1 above)
Comment (ewolinetz) [reply manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml]: @jcantrill unfortunately we may be past this update from the API perspective.. but I don't think the CLO will ever need to worry about more than one Kibana. This is leftover from the `ops` pattern. so it should only be `[0]`...

Comment (anshulvermapatel) [reply manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml]: Yes, I too think I should be only [0]

Comment (jcantrill) [reply manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml]: @ewolinetz which is fine but what happens when there are no kibana status or any of the others for that matter.  I'm trying to ensure we don't introduce some error condition

Comment (ewolinetz) [reply manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml]: @jcantrill seems to be no worse than what's already seen https://github.com/openshift/cluster-logging-operator/issues/439

**Pull request merged**